src/vs/vscode.d.ts

```diff
@@ -2510,6 +2510,18 @@ declare module 'vscode' {
 		provideDeclaration(document: TextDocument, position: Position, token: CancellationToken): ProviderResult<Declaration>;
 	}
 
+	/**
+	 * Configure the trust settings for a {@link MarkdownString}
+	 */
+	interface MarkdownStringTrustOptions {
```

Comment:
Added here for now since I couldn't figure out how to properly augment the type in `vscode.proposed.d.ts`